cmd/earth/main.go

```diff
@@ -192,13 +192,6 @@ func newEarthApp(ctx context.Context, console conslogging.ConsoleLogger) *earthA
 			Usage:       "The git password to use for git HTTPS authentication",
 			Destination: &app.gitPasswordOverride,
 		},
-		&cli.StringFlag{
-			Name:        "git-url-instead-of",
-			Value:       "https://gitlab.com/=git@gitlab.com:,https://github.com/=git@github.com:",
-			EnvVars:     []string{"GIT_URL_INSTEAD_OF"},
-			Usage:       "Rewrite git URLs of a certain pattern. Similar to git-config url.<base>.insteadOf (https://git-scm.com/docs/git-config#Documentation/git-config.txt-urlltbasegtinsteadOf). Multiple values can be separated by commas. Format: <base>=<instead-of>[,...]. For example: ''git@github.com:=https://github.com/'",
-			Destination: &app.buildkitdSettings.GitURLInsteadOf,
-		},
```

Comment:
Hmm.. this needs to be deprecated over time - we shouldn't remove it directly.

We should also add such a setting to the configuration format.

Reply:
I added this back in, and realised that I had accidentally broken it, which has been fixed in https://github.com/earthly/earthly/pull/106